BunnyJob.cs

```diff
@@ -8,13 +8,15 @@ public class BunnyJob
 	{
 		public string DisplayName;
 		public string Name;
+		public string ConsoleURL;
 		public bool Passed;
 		public List<BunnyTest> FailedTests;
 
-		public BunnyJob(string name, bool passed, List<BunnyTest> failedTests = null)
+		public BunnyJob(string name, string console_url, bool passed, List<BunnyTest> failedTests = null)
```

Comment:
nitpick: Consistent conventions: variable names should be consistent in style. Compare 'console_url' and 'failedTests'.